lib/matplotlib/axes/_base.py

```diff
@@ -3397,6 +3397,13 @@ def get_xticklabels(self, minor=False, which=None):
         -------
         ret : list
            List of `~matplotlib.text.Text` instances.
+        
+        Notes
+        -----
+        Please note that the tick label strings are not populated until a ``draw``
```

Comment:
```suggestion
        The tick label strings are not populated until a ``draw``
```
(plus re-wrapping)

Only makes it more verbose without adding information (It literally says `Notes` in the line above :smile:)